vignettes/usage.Rmd

```diff
@@ -0,0 +1,107 @@
+---
+title: "Package Usage"
+author: "Alex Gold"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Why Margins?
+For many types of regression techniques, the output of the coefficients in the
+model may not be sufficient to adequately figure out the marginal relationship
+between a covaraiate and the outcome of a regression (or the error in your estimate). In the simplest case, 
+say you run the following formula in glm `wages ~ age + age^2`. 
+
+Because the output will include coefficients for both age and age squared, it's not immediately apparent what the total marginal relationship is between a change in age and wages. Moreover, computing the error in that estimate is a non-trivial problem.
+
+This non-obviousness of marginal relationships is also a problem for even 
+very simple regressions with functional forms that mean that coefficients are
+not in the base units of the regression. For example, the coefficients of 
+logistic regression are odds ratios, so even simple regressions are not
+immediately interpretable. 
+
+This package reproduces the `margins` command from Stata, which allows for easy
+and quick estimation of marginal relationships and the associated error. The
+error is computed using the delta method, which we detail
+in a separate [vignette](delta-method.html).
+
+In this vignette, we detail a few possible use cases of the `modmarg` package.^[Modmarg is 
+short for *model margins*. It's 2 because we used a very hacky version that 
+actually ran the margins command in Stata for far too long.]
+
+# Use Case 1: Treatment Effects and Subgroup Effects
+We want to ascertain the marginal effect of `treatment` on `y` while controlling
+for age. In this first example we'll use bins of age we created earlier.
```

Comment:
"In this first example we'll use bins of age we created earlier." 

This is confusing to me since we didn't actually create anything -- do we just mean "we'll use a binned age variable"?